games-roguelike/moria/moria-5.7.10.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils
+
+DESCRIPTION="Rogue-like D&D curses game similar to nethack (BUT BETTER)"
+HOMEPAGE="https://umoria.org/"
+SRC_URI="https://github.com/dungeons-of-moria/umoria/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND=">=sys-libs/ncurses-6:0"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/umoria-${PV}"
+BUILD_DIR="${S}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_PREFIX="${EPREFIX}/usr/share"
```

Comment:
why do you want to redefine eclass-determined option? The whole `src_configure()` is useless here

Reply:
Fisrt of all `umoria` needs all these files: https://github.com/dungeons-of-moria/umoria/blob/master/src/config.cpp#L14
In the directory with the binary.
I.e. the binary is placed in `/usr/bin/umoria`, then the program will try to open `/usr/bin/data/splash.txt`, `/usr/bin/data/welcome.txt`, `/usr/bin/LICENSE` and so on.

So I place the binary into `/usr/share/umoria/` and use a wrapper to copy the binary and all these data files into `~/.local/share/umoria/`. This is the solution used in `games-roguelike/tomenet` (BTW, I have made a PR for it https://github.com/gentoo/gentoo/pull/10856)

Another solution is to patch `src/config.cpp` to make the program look for these files in more Unix-traditional directories. It's the solution used in Debian: https://sources.debian.org/src/moria/5.7.10+20181022-2/src/config.cpp/#L14. I can remake it in this way if you think it's more correct and it's worth it.

Reply:
> So I place the binary into `/usr/share/umoria/`

I mean that's why I need `-DCMAKE_INSTALL_PREFIX="${EPREFIX}/usr/share"`

Reply:
I would prefer a patch and/or sed or if it's really awkward, put it in `/opt`. We try to keep `/usr/share` platform-neutral as much as possible.

Reply:
@chewi Thank you for the comment. I'll remake the ebuild then.

Will it be acceptable to put `LICENSE` file into `/usr/share/doc/${P}` or anywhere else?
If we don't install `LICENSE` at all, the game will show an error when it will try to open it, but it won't affect the game itself.